app/mailers/alert_mailer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Mailer for sending alerts related to ETD submissions
+class AlertMailer < ApplicationMailer
+  def ready_for_cataloging
+    # ETDs loaded since yesterday morning
+    @etds_uncataloged_new = Submission.ils_records_created_since_yesterday_morning.order(:folio_instance_hrid)
+    # All ETDs awaiting full cataloging
+    @etds_uncataloged_all = Submission.at_ils_loaded.where.not(druid: Settings.skip_cataloging_alert)
+                                      .order(:folio_instance_hrid)
+
+    return if @etds_uncataloged_all.empty?
+
+    send_message(subject: "#{subject_prefix} ETDs ready to be cataloged",
+                 template_name: 'ready_for_cataloging',
+                 to_address: Settings.etd_cataloging_list)
+  end
+
+  private
+
+  def subject_prefix
+    "[#{Settings.ps_env.upcase}]"
+  end
+
+  def send_message(subject:, template_name:, to_address: nil)
+    return unless alertable_environment?
+
+    to_address ||= self.class.default[:to]
+    mail(subject:, template_name:, to: to_address)
+  end
+
+  def alertable_environment?
+    Settings.ps_env.in?(Settings.alertable_environments)
+  end
```

Comment:
Instead of relying on custom Settings-based ways of handling envs, heracles has been leaning on honeybadger. So how about something like:

```suggestion
    return unless environment.in?(Settings.mailable_environments)

    # NOTE: Also, change default `to` address in ApplicationMailer to Settings.etd_cataloging_list
    mail(subject: "#{[#{environment.upcase}]} ETDs ready to be cataloged", 
         template_name: 'ready_for_cataloging')
  end

  private

  def environment
    Honeybadger.config[:env]
  end
```

Reply:
I like it. I was starting to think about a follow up refactor because there seemed to be a lot of unnecessary abstraction here, so yeah I think it.